cores/esp32/HardwareSerial.cpp

```diff
@@ -114,37 +114,46 @@ void serialEventRun(void)
 }
 #endif
 
-
 HardwareSerial::HardwareSerial(int uart_nr) : _uart_nr(uart_nr), _uart(NULL), _rxBufferSize(256) {}
 
+enum { UART_NUM_0, UART_NUM_1, UART_NUM_2 };
```

Comment:
If this enum will be used just for the switch statement, let's use int instead

Reply:
It is solved now by including "driver/uart.h" that has those defines.